packages/itwinui-react/src/core/SideNavigation/SideNavigation.tsx

```diff
@@ -151,14 +151,14 @@ export const SideNavigation = React.forwardRef((props, forwardedRef) => {
         {expanderPlacement === 'bottom' && ExpandButton}
       </Box>
       {submenu && (
-        <WithCSSTransition
-          in={isSubmenuOpen}
-          dimension='width'
-          timeout={200}
-          classNames='iui'
+        <Box
```

Comment:
any chance we can avoid the extra div? there was already a wrapping div before, so now there are two.

Reply:
do you mean the wrapper for both the side nav bar and the submenu?

Reply:
no, there are two divs around the submenu content
<img width="624" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/7da2a7d3-b427-4c2b-990f-4dbe38a8304c">

Reply:
oh, maybe the wrapper is needed for resizing 🤔 
https://github.com/iTwin/iTwinUI/assets/9084735/6a2f3bf6-26fa-4e22-90c4-cb37c5672a34

Reply:
now that you mention this, I notice it seems to be an issue even for the original version (maybe)
Original version, when you resize then retract the submenu, then expand again, it doesn't keep the previously resized size. Is that intended? Probably can't be done differently without JS
![sidenavog](https://github.com/iTwin/iTwinUI/assets/22462084/5b7179c4-172d-42b0-acc7-fc7357fae638)

and also an issue for the current solution (with the extra div):
![sidenavgrid](https://github.com/iTwin/iTwinUI/assets/22462084/a8b89ae5-a34b-40a9-8b71-5b7377a457b5)

which I think is due to grid being fixed after resizing?

Reply:
> Original version, when you resize then retract the submenu, then expand again, it doesn't keep the previously resized size.

imo this is ok. the size is temporary

> and also an issue for the current solution (with the extra div)
> which I think is due to grid being fixed after resizing?

this is not ok. it should not occupy any space after closing. for now, maybe you could use some js to remove the `width` style when closing.

Reply:
is there a way to do that without adding an extra prop to pass `isSubmenuOpen` to submenu?